src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
```

Comment:
Well, I'm suggesting that it be inside a .Social namespace.  I don't think the .Web part was necessary...I think that's what it defaulted when the first class was created :-)  But if you keep the .Web for others then I would think that you also keep the .Web for this....so .Web.Social.ExtensionData